ctapipe/image/ellipsoid.py

```diff
@@ -0,0 +1,443 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: UTF-8 -*-
+"""
+Shower image parametrization based on image fitting.
+"""
+
+from enum import Enum
+
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import Angle
+from iminuit import Minuit
+
+from ctapipe.image.cleaning import dilate
+from ctapipe.image.hillas import hillas_parameters
+from ctapipe.image.pixel_likelihood import (
+    mean_poisson_likelihood_gaussian,
+    neg_log_likelihood_approx,
+)
+from ctapipe.image.toymodel import SkewedGaussian, SkewedGaussianLaplace
+
+from ..containers import ImageFitParametersContainer
+
+PED_TABLE = {
```

Comment:
Where do these come from? Why these keys?

Reply:
I do not think the pedestal width is available at the moment. I used those values that I found in Dan's code as he also had to load the pedestal width like that.